paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -901,5 +887,42 @@ static void SetDstMemoryHandler(
   (*dst_memory_p)->set_data_handle(to_void_cast<T>(output_data));
 }
 
+template <typename T>
+static void SetDstMemoryQuantized(
+    const framework::ExecutionContext& ctx, framework::Tensor* output,
+    std::vector<int> dst_tz, const mkldnn::engine& engine,
+    std::shared_ptr<mkldnn::memory::primitive_desc>& dst_pd,  // NOLINT
+    std::shared_ptr<mkldnn::memory>& dst_memory) {            // NOLINT
+  T* output_data = output->mutable_data<T>(ctx.GetPlace());
+  const size_t dst_dims = dst_tz.size();
+  memory::format dst_fmt;
+  PADDLE_ENFORCE(dst_dims <= 5,
+                 "Dst memory for quantization can not have dims > 5");
+  switch (dst_dims) {
```

Comment:
Please use `platform::MKLDNNFormatForSize` from `mkldnn_helper.h`.

Reply:
since quantize CONV-OP prefer `nwc`/`ndhwc` when `dim-size` equal to `3/5`, so this part has changed to like below:
```cpp
  dst_fmt = (dst_dims == 4)
                ? memory::format::nhwc
                : platform::MKLDNNFormatForSize(dst_dims, memory::format::nhwc);
```